src/entities/profile/setting-profile.entity.ts

```diff
@@ -0,0 +1,26 @@
+import { Column, Entity, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { ProfileEntity } from './profile.entity';
+
+@Entity('setting')
```

Comment:
I dont think we need it, we have user table

Reply:
It's not working before not saving data to database and I'm created new entity it's like for freelancer